x-pack/test/reporting_api_integration/reporting_and_security/error_codes.ts

```diff
@@ -6,6 +6,7 @@
  */
 
 import expect from '@kbn/expect';
+import { INTERNAL_ROUTES } from '@kbn/reporting-plugin/common/constants/routes';
```

Comment:
For some reason, I can't simply import from `@kbn/reporting-plugin/common/constants`, may be due to a different typescript config in tests vs the plugin code.